Build not working in private scope

Logs from:  ng build --configuration production.private

[INFO] Generating flattened POM of project pl.cyfronet.s4e:s4e-web:pom:4.0.0-SNAPSHOT...
[INFO] 
[INFO] --- frontend-maven-plugin:1.8.0:npm (npm test) @ s4e-web ---
[INFO] Skipping execution.
[INFO] 
[INFO] --- frontend-maven-plugin:1.8.0:npm (npm run-script build) @ s4e-web ---
[INFO] Running 'npm run-script build-private' in /app/s4e-web
[INFO] 
[INFO] > s4e-angular@0.0.2 build-private /app/s4e-web
[INFO] > ng build --configuration production.private
[INFO] 
[INFO] 
[INFO] Date: 2020-02-25T20:18:14.097Z
[INFO] Hash: 032bae1f0fca90ade318
[INFO] Time: 19795ms
[INFO] chunk {0} runtime.06daa30a2963fa413676.js (runtime) 1.44 kB [entry] [rendered]
[INFO] chunk {1} main.9868d9b237c3a48c54da.js (main) 128 bytes [initial] [rendered]
[INFO] chunk {2} polyfills.85f47f0bf59079cbc23a.js (polyfills) 130 bytes [initial] [rendered]
[INFO] chunk {3} styles.f3267c8ecd55611fc51a.css (styles) 89 kB [initial] [rendered]
[INFO] 
[INFO] ERROR in src/app/views/map-view/zk/report-modal/report-modal.component.html(29,33): : Argument of type '"maxlength"' is not assignable to parameter of type '"max" | "required" | "pattern" | "min" | "requiredTrue" | "email" | "minLength" | "maxLength" | "fileRequired" | "filesMinLength" | "filesMaxLength" | "fileMaxSize"'.
[INFO] src/app/views/map-view/zk/report-modal/report-modal.component.html(35,33): : Argument of type '"maxlength"' is not assignable to parameter of type '"max" | "required" | "pattern" | "min" | "requiredTrue" | "email" | "minLength" | "maxLength" | "fileRequired" | "filesMinLength" | "filesMaxLength" | "fileMaxSize"'.
[INFO] 
[INFO] npm ERR! code ELIFECYCLE
[INFO] npm ERR! errno 1
[INFO] npm ERR! s4e-angular@0.0.2 build-private: `ng build --configuration production.private`
[INFO] npm ERR! Exit status 1
[INFO] npm ERR! 
[INFO] npm ERR! Failed at the s4e-angular@0.0.2 build-private script.
[INFO] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -25,8 +25,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Build in private scope [#372](https://github.com/cyfronet-fid/sat4envi/pull/372)
 - Constraints names on Product and Scene tables [#364](https://github.com/cyfronet-fid/sat4envi/pull/364)
-- Build in private scope
 
 ### Updates
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Constraints names on Product and Scene tables [#364](https://github.com/cyfronet-fid/sat4envi/pull/364)
+- Build in private scope
 
 ### Updates
 

--- a/s4e-web/package.json
+++ b/s4e-web/package.json
@@ -7,7 +7,7 @@
     "start-private": "ng serve --proxy-config proxy.conf.json --configuration hmr.private",
     "start-production": "ng serve --proxy-config proxy.conf.json --configuration production",
     "start-production-private": "ng serve --proxy-config proxy.conf.json --configuration production.private",
-    "build": "ng build",
+    "build": "ng build --prod",
     "build-private": "ng build --configuration production.private",
     "test": "ng test",
     "lint": "ng lint",

--- a/s4e-web/src/app/views/map-view/zk/report-modal/report-modal.component.html
+++ b/s4e-web/src/app/views/map-view/zk/report-modal/report-modal.component.html
@@ -26,13 +26,13 @@
             <input id="caption" formControlName="caption" type="text" />
             <span class="error" *ngIf="form.controls.caption.hasError('required') && form.controls.caption.touched" i18n>To pole jest wymagane</span>
             <!--suppress TypeScriptValidateTypes -->
-            <span class="error" *ngIf="form.controls.caption.hasError('maxlength') && form.controls.caption.touched" i18n>Maksymalna liczba znaków: 80</span>
+            <span class="error" *ngIf="form.controls.caption.hasError('maxLength') && form.controls.caption.touched" i18n>Maksymalna liczba znaków: 80</span>
           </li>
           <li>
             <label for="notes" i18n>Notatka</label>
             <textarea id="notes" formControlName="notes"></textarea>
             <!--suppress TypeScriptValidateTypes -->
-            <span class="error" *ngIf="form.controls.notes.hasError('maxlength') && form.controls.notes.touched" i18n>Maksymalna liczba znaków: 800</span>
+            <span class="error" *ngIf="form.controls.notes.hasError('maxLength') && form.controls.notes.touched" i18n>Maksymalna liczba znaków: 800</span>
           </li>
         </ul>
       </div>

